docs/application/native/guides/machine-learning/machine-learning-service.md

```diff
@@ -192,10 +196,111 @@ Clients (application developers) can use services with the Machine Learning Serv
+
+// Parameter deciding whether to activate this model or not
+const bool is_active = true;
+
+// Model description parameter
+const gchar *description = "This is description of the mobilenet_v2 model ...";
+
+// Out parameter
+unsigned int version;
+
+// Register the model via ML Service API
+ml_service_model_register (key, improved_model_path, is_active, description, &version);
+
+// Update the model description
+ml_service_model_update_description (key, version, "Updated description for mobilenet_v2");
+
+...
+
+// Register a sample model as NOT active
+ml_service_model_register (key, "/some/path/to/model.file", false, "not yet active", &version);
+
+// Get model info with its name and version
+ml_option_h model_info;
+ml_service_model_get (key, version, &model_info);
+
+// Check values of retrieved model info
+gchar *_path;
+ml_option_get (model_info, "path", (void **) &_path); // Value of _path: "/some/path/to/model.file"
+gchar *_description;
+ml_option_get (model_info, "description", (void **) &_description); // Value of _description: "not yet active"
+
+// Activate the not yet active model, setting all other models as NOT active
+ml_service_model_activate (key, version);
+
+// Get all model info with same name
+ml_option_h *all_info_list;
+guint info_num;
+ml_service_model_get_all (key, &all_info_list, &info_num);
+
+for (int i = 0; i < info_num; i++) {
+  // Do something for each model info
+  ...
+}
+
+// Delete the model from the ml service
+ml_service_model_delete (key, version);
+```
+
+### Application code using the registered model
+
+Application developers can use the registered model:
+
+```c
+// The name shared with AI developers
+const gchar *key = "imgcls-mobilenet";
+gchar *model_path;
+
+ml_option_h activated_model_info;
+
+// Get registered and activated model via ML Service API
+int status = ml_service_model_get_activated (key, &activated_model_info);
+if (status == ML_ERROR_NONE) {
+  // If there is a model registered by AI developers, use it
+  gchar *activated_model_path;
+
+  // Get path of the model
+  status = ml_option_get (activated_model_info, "path", (void **) &activated_model_path);
+
+  model_path = g_strdup (activated_model_path);
+} else {
+  // If there is no registered model, use the default model in the app
+  model_path = g_strdup_printf ("%s/%s", app_get_resource_path (), "mobilenet_v1.tflite");
+}
+
+// Do inference with the variable `model_path`
+...
+
+```
+
 ## Related information
 
 - Dependencies
-  - Tizen 7.0 and Higher for Mobile
-  - Tizen 7.0 and Higher for Wearable
-  - Tizen 7.0 and Higher for TV
-  - Tizen 7.0 and Higher for IoT
+
+    Machine Learning Service API for Pipeline
+    - Tizen 7.0 and Higher for Mobile
+    - Tizen 7.0 and Higher for Wearable
+    - Tizen 7.0 and Higher for IoT
+
+    Machine Learning Service API for Model
+    - Tizen 7.5 and Higher for Mobile
+    - Tizen 7.5 and Higher for Wearable
+    - Tizen 7.5 and Higher for IoT
```

Comment:
For dependencies, it is recommended to mention the lowest of the supported version.
For model, it would be better to use an alert to mention the version of support.
```suggestion
```